eservice/pdo/eservice/pdo_enclave.py

```diff
@@ -192,6 +193,21 @@ def get_enclave_basename():
     global _pdo
     return _pdo.basename if _pdo is not None else None
 
+# -----------------------------------------------------------------
+# -----------------------------------------------------------------
+
+def get_enclave_epid_group():
+    global _epid_group
+    _epid_group = _pdo.get_epid_group()
+    return _epid_group
+
+# -----------------------------------------------------------------
+# -----------------------------------------------------------------
+
+def dump_enclave_ias_settings():
```

Comment:
New function get_enclave_epid_group() doesn't seem to be used. Is it an omission or it is intended for the future use?

Reply:
My apologies - wrong copy and paste. I meant dump_enclave_is_settings()

Reply:
It is used in the eservice-enclave-info script.